### PR TITLE
Adding test for taskqueue pull worker. Fixes #254

### LINKED ISSUE #254: appengine/taskqueue/pull-counter Needs Worker Tests
The app in appengine/taskqueue/pull-counter has a worker that is run as a separate module using the worker.yaml. It listens on /_ah/start and then enters an infinite loop pulling from the pull taskqueue and pushing to datastore.

Currently the test just tests the post and addition to the queue, but not the worker function.

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: appengine/taskqueue/pull-counter/pullcounter_test.py [hunk 1]
```diff
@@ -31,3 +32,11 @@ def test_app(testbed):
     tasks = tq_stub.get_filtered_tasks()
     assert len(tasks) == 1
     assert tasks[0].name == 'task1'
+
+    with mock.patch('main.update_counter') as mock_update:
+        # Force update to fail, otherwise the loop will go forever.
```
Comment (jerjou) [inline appengine/taskqueue/pull-counter/pullcounter_test.py]: I'm probably missing some context, but is it not weird to have a worker that polls indefinitely? I'd expect a cron job or something instead, which you can at least cancel?

**Comment by jerjou** [comment]
LGTM

**Comment by waprin** [comment]
lgtm

**Comment by jonparrott** [comment]
Travis is being exceptionally slow today. I verified locally.

**Pull request merged**